eng/Packages.Data.props

```diff
@@ -134,8 +134,8 @@
     <PackageReference Update="Microsoft.Azure.SignalR.Management" Version="1.18.0" />
     <PackageReference Update="Microsoft.Azure.SignalR.Protocols" Version="1.18.0" />
     <PackageReference Update="Microsoft.Azure.SignalR.Serverless.Protocols" Version="1.9.0" />
-    <PackageReference Update="Microsoft.Azure.WebJobs" Version="3.0.33" />
-    <PackageReference Update="Microsoft.Azure.WebJobs.Sources" Version="3.0.30" />
+    <PackageReference Update="Microsoft.Azure.WebJobs" Version="3.0.35" />
+    <PackageReference Update="Microsoft.Azure.WebJobs.Sources" Version="3.0.35" />
```

Comment:
Want to make sure 3.0.35 is correct or if we should be using `3.0.35-11986` 

@mathewc - thoughts?

Reply:
The Azure SDK packages can only reference official builds of these packages, so it'll need to be 3.0.35 or whatever the latest version is with the required changes